Dockerfile

```diff
@@ -1,62 +1,56 @@
-FROM python:3.10-slim
+FROM nvidia/cuda:12.0.1-base-ubuntu22.04
```

Comment:
Our Dockerfile needs to ensure that the container has the right dependencies in place to support all of the different types of hardware acceleration that exist in Squishy. In this case, we are adding support for CUDA, but we need to be sure that we don't lose support for VAAPI